### ISSUE TITLE
Indicate Unsupported Filters

### ISSUE THREAD

**Issue by Nzzyme** [AdguardTeam/AdguardForAndroid#5512]
### Issue Details

I added a [custom IP filter](https://[raw.githubusercontent.com/stamparm/ipsum/refs/heads/master/ipsum.txt](https://raw.githubusercontent.com/stamparm/ipsum/refs/heads/master/ipsum.txt)) by navigating to:

Settings > Filtering > Filters > Custom filters

Later, I updated all the filters from the main home screen. When I opened a web browser and tried to open one of the IP addresses listed in the filter link (For Ex: 195.158.4.210), AG did not block this in the web browser perhaps due to unsupported filter rule syntax. Checking the 'Recent activity' confirmed no block on that IP address. 

### Proposed solution

In this case, it would be beneficial if AG immediately notifies the user after adding a filter that the filter is unsupported (displaying "Filter not supported" text in red) and prevents the user from enabling such a filter (by graying out the toggle switch for unsupported filters). Currently, the filter can be enabled, giving the impression that everything is 'fine' with that filter, when in reality, it is not effective.

![image](https://github.com/user-attachments/assets/8172f1d5-2820-4bc9-9079-06f6e5433a69)

### Alternative solution

Manually check the compatibility of filters

**Comment by Nzzyme** [comment]
> @Nzzyme unfortunately it is not possible to open or copy the filter link. Could you please re-send the link or email [qa@adguard.com](mailto:qa@adguard.com) with issue number 5512?

Here you go:

[https://[raw.githubusercontent.com/stamparm/ipsum/refs/heads/master/ipsum.txt](https://raw.githubusercontent.com/stamparm/ipsum/refs/heads/master/ipsum.txt)](url)

**Comment by maxikuzmin** [comment]
@Nzzyme thank you, we'll think about how this can be improved and will improve it in the next updates

**Comment by ameshkov** [comment]
I don't think it makes sense to invest time into this. There's no clear way to distinguish compatible and incompatible filters given how complex the syntax is.
